services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/permissions/PermissionServiceITest.kt

```diff
@@ -20,17 +21,19 @@ import org.springframework.security.test.context.support.WithMockUser
 import org.springframework.test.context.ActiveProfiles
 import org.testcontainers.junit.jupiter.Testcontainers
 
+private const val HANKE_TUNNUS = HankeFactory.defaultHankeTunnus
+private const val USERNAME = "user"
```

Comment:
This pattern is mostly used to notate the current user, i.e. the user that's used as `@WithMockUser`. Here, the user needs to be different, so this pattern might not be optimal.

Reply:
Fixed.